neqo-transport/src/pace.rs

```diff
@@ -78,6 +86,12 @@ impl Pacer {
     /// an estimate of the round trip time (`rtt`), the estimated congestion
     /// window (`cwnd`), and the number of bytes that were sent (`count`).
     pub fn spend(&mut self, now: Instant, rtt: Duration, cwnd: usize, count: usize) {
+        if !self.enabled {
+            self.c = self.m;
```

Comment:
```suggestion
```

The value of `self.c` is assigned to `m` and doesn't change.

If you were to add a `set_enabled(bool)` function, then you would want to have that function set `self.c` like this when disabling pacing, as you do in the constructor.  But you don't need to do it every time.